src/main/cms/cms.c

```diff
@@ -984,7 +984,7 @@ static void cmsTraverseGlobalExit(const CMS_Menu *pMenu)
 
 const void *cmsMenuExit(displayPort_t *pDisplay, const void *ptr)
 {
-    int exitType = (int)ptr;
+    intptr_t exitType = (intptr_t)ptr;
```

Comment:
IMO this is more explicit:
```suggestion
    int exitType = (intptr_t)ptr;
```

Reply:
Also see https://github.com/betaflight/betaflight/blob/b9b4b77e3b0a94fd65754c28aa194f6e50c85fea/src/main/cli/cli.c#L1566